addons/skin.estuary/xml/View_53_Shift.xml

```diff
@@ -290,6 +121,68 @@
 			</control>
 		</control>
 	</include>
+	<include name="ShiftImageBox">
+		<control type="image">
+			<left>18</left>
+			<top>90</top>
+			<width>334</width>
+			<height>480</height>
+			<aspectratio aligny="center">stretch</aspectratio>
+			<texture>dialogs/dialog-bg-nobo.png</texture>
+			<bordertexture border="21" infill="false">overlays/shadow.png</bordertexture>
+			<bordersize>20</bordersize>
+			<visible>String.IsEmpty(ListItem.Art(thumb)) + !Container.Content(albums) | String.IsEmpty(ListItem.Art(poster)) + [Container.Content(movies) | Container.Content(tvshows)]</visible>
```

Comment:
Due to below changes for the poster/thumb images needed to add additional condition to prevent this being shown when Shift view is selected for Albums.